src/dealWizard/openProposalWizard/openProposalProposalLeadStage/openProposalProposalLeadStage.ts

```diff
@@ -1,33 +1,39 @@
 import { IStageMeta } from "./../../dealWizardTypes";
 import { autoinject } from "aurelia-framework";
 import { IBaseWizardStage } from "../../dealWizardTypes";
-import { WizardService, IWizardState, WizardErrors } from "../../../services/WizardService";
+import { IWizardState, WizardErrors, WizardService } from "../../../services/WizardService";
 import { IDealRegistrationTokenSwap, IProposalLead } from "entities/DealRegistrationTokenSwap";
+import { validateTrigger, ValidationController, ValidationControllerFactory } from "aurelia-validation";
+import { PrimeRenderer } from "../../../resources/elements/primeDesignSystem/validation/renderer";
+import { getErrorsFromValidateResults, proposalLeadValidationRules } from "../../validation";
 
 @autoinject
 export class OpenProposalProposalLeadStage implements IBaseWizardStage {
   public wizardManager: any;
   public wizardState: IWizardState<IDealRegistrationTokenSwap>;
   public errors: WizardErrors<IProposalLead> = {};
+  public form: ValidationController;
 
-  constructor(public wizardService: WizardService) {}
+  constructor(public wizardService: WizardService, validationFactory: ValidationControllerFactory) {
+    this.form = validationFactory.createForCurrentScope();
```

Comment:
can we make this automatically happen for all stages by moving it into the WizardService?

Reply:
I moved this in the proposalLeadSection component